### ISSUE TITLE
install.sh fails (dbus-python?) on 2 of 3 Ubuntu 16.04 Unity

### ISSUE THREAD

**Issue by MLXXXp** [bluppfisk/coinprice-indicator#65]
I have three systems running Ubuntu 16.04 with Unity 7, all set up fairly similarly. On the first system, _install.sh_ runs without errors and the indicator works fine using _run.sh_. On the other two systems, _install.sh_ fails in the same way. The last line of console output is:
`./install.sh: line 6: vext: command not found`
(so this may be a duplicate of issue #63)

Between the working system and the failing ones, the _install.sh_ output is nearly identical up until the line:
`  Running setup.py install for dbus-python ...`
where the line ends with `done` for the good system and `error` for the failing ones.

After this error, the output contains some detailed info about the problem. However, I'm not familiar enough with Python to figure out what the problem is, especially when using a _virtualenv_.

I've attached 3 files containing the console output from running _install.sh_ on each system:

[install_out_sys1_good.txt](https://github.com/bluppfisk/coinprice-indicator/files/3343144/install_out_sys1_good.txt) (successful build)
[install_out_sys2_fail.txt](https://github.com/bluppfisk/coinprice-indicator/files/3343145/install_out_sys2_fail.txt)
[install_out_sys3_fail.txt](https://github.com/bluppfisk/coinprice-indicator/files/3343146/install_out_sys3_fail.txt)

**Comment by bluppfisk** [comment]
Hi 

Have you tried doing a pip install wheel in the virtual environment? My impression from reading the logs is that you may not have that installed (hence the wheel building that fails). You may also need to upgrade pip to the latest version (also inside the virtual env) or even install the python dev package.

Would appreciate you trying the above (maybe even step by step) and trying to rerun the installer afterward to see what solves the problem. It is indeed possible that I take a certain package for granted that isn't there on every system.

**Comment by MLXXXp** [comment]
Problem solved!

Upgrading _pip_ to the latest version or installing _wheel_, in the virtual environment, didn't help.

I turns out that two Ubuntu packages were missing.

When I saw the error about installing the development version of Python, I verified that package _python&#8209;dev_ had been installed. I didn't clue in to the fact that I actually needed package
**python3-dev**

Once _python3-dev_ was installed, I got the message:

```text
    checking for DBUS... no
    configure: error: Package requirements (dbus-1 >= 1.8) were not met:
    
    No package 'dbus-1' found
```

This was solved by installing package
**libdbus-1-dev**
